src/uct/ib/base/ib_iface.c

```diff
@@ -219,6 +219,20 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
   {NULL}
 };
 
+#ifdef ENABLE_STATS
+static ucs_stats_class_t uct_ib_iface_stats_class = {
+    .name          = "ib_iface",
+    .num_counters  = UCT_IB_IFACE_STAT_LAST,
+    .class_id      = UCS_STATS_CLASS_ID_INVALID,
+    .counter_names = {
+        [UCT_IB_IFACE_STAT_RX_COMPLETION]        = "rx_completion",
+        [UCT_IB_IFACE_STAT_TX_COMPLETION]        = "tx_completion",
+        [UCT_IB_IFACE_STAT_RX_COMPLETION_ZIPPED] = "rx_completion_zipped",
+        [UCT_IB_IFACE_STAT_TX_COMPLETION_ZIPPED] = "tx_completion_zipped"
```

Comment:
don't we need it only for mlx5 iface?

Reply:
Yes, we need it only for mlx5 iface. But we don't have common `uct_ib_mlx5_iface_t` iface for this purpose. And introducing the separate interface for this seems like overkill from my point of view.

The first implementation contains `mlx5_` prefix for these counter, but later it was deleted. The motivation of removing this prefix is that in case of using verbs API we cannot observe CQE zipping, so there is no need to highlight relations to mlx5 iface only.

But if we want to highlight relation of these counters to `mlx5_`, then I can propose two ways to do that:
1) Just add `mlx5_` prefix to counters
2) Add `mlx5_stats` statistic field to `uct_ib_iface_t` struct and initialize this counters inside this field.